Gruntfile.js

```diff
@@ -295,6 +306,11 @@ module.exports = function (grunt) {
         "yuidoc"
     ]);
 
+    grunt.registerTask("release", [
+        "build",
+        "bump"
+    ]);
+
```

Comment:
you have 2 `release` tasks in this gruntfile, i think this one would get overwritten